onebusaway-android/src/main/res/values/donottranslate.xml

```diff
@@ -61,6 +61,8 @@
     <string name="preference_key_social_category">preference_category_social</string>
     <string name="preference_key_social">preference_social</string>
     <string name="preference_key_never_show_payment_warning_dialog">never_show_payment_warning_dialog</string>
+    <string name="preferences_key_user_share_logs">preferences_user_share_logs</string>
+    <string name="preferences_key_user_logs">Share trip logs</string>
```

Comment:
You'll need to split this into two strings - the one in `donottranslate.xml` should have the underscores to match the conventions for the other strings, but the one that's used for the preference category title (that the user sees) needs to be in `strings.xml` so it can be translated into other languages.

Reply:
@barbeau I have split into two strings to match the conventions. Please review and let me know if it looks good.